packages/@vuepress/theme-default/components/Navbar.vue

```diff
@@ -125,4 +125,9 @@ $navbar-horizontal-padding = 1.5rem
       display none
     .links
       padding-left 1.5rem
+    .site-name
+      width calc(100vw - 9.4rem)
```

Comment:
Where does the `9.4rem` come from? A comment would be helpful because it seems custom otherwise

Reply:
this value is caculated by padding and the width of search icon

![image](https://user-images.githubusercontent.com/19368146/64476163-24a07e00-d1be-11e9-8b49-a0fca3132fbe.png)

padding-left: 4.5rem
padding-right: 1.5rem
sum: 4.5 + 1.5 = 6rem

![image](https://user-images.githubusercontent.com/19368146/64476186-721ceb00-d1be-11e9-8d49-b8d772984bee.png)

padding-left: 1.5rem
width: 2rem
sum: 2 + 1.5 = 3.5rem

so `.site-name width` = 100vw - (6rem + 3.5rem) = 100vw - 9.5rem

but why 9.4rem ？Because I found that it could display one more Chinese character in the iPhone 6/7/8 Plus (Chinese only)

100vw -9.5rem
![image](https://user-images.githubusercontent.com/19368146/64476443-2455b200-d1c1-11e9-80ae-0f39fa381f0b.png)

100vw - 9.4rem
![image](https://user-images.githubusercontent.com/19368146/64476441-1f90fe00-d1c1-11e9-9640-c3b0b5ba147e.png)

I think the title should show as much text as possible.

Reply:
@bencodezen is that ok for you?

Reply:
@kefranabg Yup I'm good for now. In the future, we may want to abstract out these calculations since these are "magic numbers," but since the issue number is tagged in the commit, let's go ahead and merge.